torch/backends/xeon/run_cpu.py

```diff
@@ -311,6 +311,7 @@ def is_numactl_available(self):
                 env=os.environ,
                 stdout=subprocess.DEVNULL,
                 stderr=subprocess.DEVNULL,
+                check=True,
```

Comment:
`returncode` is checked on the next line, isn't it?
```suggestion
                check=False,
```